ptf

```diff
@@ -133,6 +133,7 @@ be subtracted from the result by prefixing them with the '^' character.  """
             # Parse --interface
             def check_interface(value):
                 try:
```

Comment:
what does try statement handle in next 3 lines?
If nothing, next 3 lines must no be under the try statement.

Reply:
There is no reason to return any interface configuration in case values haven't been parsed correctly. So the script should be stopped.

Reply:
Could you please provide case that will be handled by this try statement in  next 4 lines
```
                    port_info = ""
                    sp = ';'
                    if sp in value:
                        value, port_info = value.split(sp, 1)
```
Seems that it doesn't handle next 4 lines and this lines can be moved out of the try scope, it simplifies code investigation, because shrinks possible rows to check for exception.

Reply:
Done